openpgp/v2/write.go

```diff
@@ -594,8 +594,8 @@ func encrypt(
 	encryptKeys := make([]Key, len(to)+len(toHidden))
 
 	config := params.Config
-	// AEAD is used only if config enables it and every key supports it
-	aeadSupported := config.AEAD() != nil
+	// AEAD is used if every key supports it
+	aeadSupported := true
```

Comment:
I think since it's just a one-line change, it shouldn't be too much effort to carry a patch; not sure it warrants a config if nobody else needs it :relieved: